### PR TITLE
Adds context to ksonnet guide and fixes various references

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes issue #152

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/153)

### CODE REVIEW

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This documentation assumes you have a Kubernetes cluster already available.
 
   * For local environment try [Minikube setup](/docs/started/getting-started-minikube/).
   * For cloud environment try [GKE setup](/docs/started/getting-started-gke/).
-  * For general k8s cluster try [quick start](/docs/started/getting-started/#quick-start) or [install via ksonnet](/docs/started/getting-started/#install-via-ksonnet).
+  * For general k8s cluster try [quick start](/docs/started/getting-started/#quick-start) or [install via ksonnet](/docs/guides/components/ksonnet).
```
Comment (abhi-g) [inline content/docs/started/getting-started.md]: Actually,  this line isn't the correct recommendation anymore. We can remove this. The rest of the doc as upated by Jeremy in #158 would provide an easy setup on existing k8s cluster.

Comment (sarahmaddox) [reply content/docs/started/getting-started.md]: I've merged with the latest content in the repo and removed this bullet item.

**Pull request merged**